src/styles/blocks/catalog.scss

```diff
@@ -17,6 +17,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all 300ms ease;
+    color: black;
+
+    &:hover {
+      transform: scale(1.2);
+      color: #34568b;
```

Comment:
Consider using a SCSS variable for the color `#34568b`, e.g., `$card-title-hover-color: #34568b;` and then use it like `color: $card-title-hover-color;`.